recipes/comet-ms/2023010/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+make CXX=${CXX} CXXFLAGS="${CXXFLAGS} -mcmodel=large -fpic -shared"
```

Comment:
static probably not needed because of:
https://github.com/UWPR/Comet/blob/master/Makefile#L9

Reply:
```suggestion
make CXX=${CXX} CXXFLAGS="${CXXFLAGS} -mcmodel=large -fpic" CFLAGS="${CFLAGS} -mcmodel=large -fpic"
```